src/data/openings.json

```diff
@@ -417,5 +417,49 @@
       "Excellent organizational and follow-up skills",
       "Strong math skills"
     ]
-  }
+  },
+{
+ "openingId": "technical-project-manager",
+    "applicationMethods": [
+      {
+        "channel": "slack",
+        "note": "Message [Justin]"
+      },
+      {
+        "channel": "linkedin"
+      },
+      {
+        "channel": "twitter"
+      },
+      {
+        "channel": "reddit"
+      }
+    ],
+    "category": "Engineering",
+    "description": "Responsible for providing project planning and resource management and ensuring that projects are completed to specification. Provide expertise and strategic advice on best practices and leverage change management principles to ensure project success.",
+    "payNotes": ["2,800 coins/day"],
+    "position": "Technical Project Manager",
+    "reportsTo": [
+      {
+        "githubUsername": "Angle943",
```

Comment:
It doesn't make any difference in navigation, but the username is all lowercase: `angle943`